internal/db/schema/migrations/oss/postgres/99/03_worker_authentication.up.sql

```diff
@@ -40,7 +57,10 @@ create table worker_auth_ca_certificate(
    constraint worker_auth_operational_indicator_enm_fkey
      references worker_auth_operational_indicator_enm(state)
        on delete restrict
-       on update cascade
+       on update cascade,
+  issuing_ca text not null
```

Comment:
should this be `wt_private_id`?

Reply:
The CA id will only ever be "roots", which isn't long enough for the wt_private_id length constraint